pdns/backends/gsql/gsqlbackend.cc

```diff
@@ -1502,6 +1502,32 @@ bool GSQLBackend::replaceComments(const uint32_t domain_id, const string& qname,
   return true;
 }
 
+string GSQLBackend::directGSQLQuery(const string &query)
+{
+ try {
+   ostringstream out;
+
+   unique_ptr<SSqlStatement> stmt(d_db->prepare(query,0));
```

Comment:
@cmouse can you confirm that just calling the destructor on SSqlStatement when it goes out of scope is sufficient, without calling `reset` or `release`?